packages/react/config/jest/setup.js

```diff
@@ -44,14 +49,15 @@ if (typeof window !== 'undefined') {
   window.HTMLElement.prototype.scrollIntoView = jest.fn();
 }
 
-// Needed so that any component that uses sizeme can be jest tested
-import sizeMe from 'react-sizeme';
-import dayjs from '../../src/utils/dayjs';
-
 sizeMe.noPlaceholders = true;
 
 // Force the timezone to be the same everywhere
 dayjs.tz.setDefault('America/Chicago');
 Date.prototype.getTimezoneOffset = () => 300; // mock date offset
 MockDate.set(1537538254000);
 Date.prototype.getLocaleString = () => 'Mock Date!';
+
+// must check for window first or SSR tests will fail
+if (typeof window !== 'undefined' && typeof window.URL.createObjectURL === 'undefined') {
+  Object.defineProperty(window.URL, 'createObjectURL', { value: function noop() {} });
+}
```

Comment:
this mocking of createObjectURL is setup here to allow it to be used in StorySnapshot tests, otherwise warnings are thrown about this function not existing in Jest--even if it is mocked in beforeAll/beforeEach of the snapshot test file.